libbeat/docs/release-notes/breaking/breaking-7.7.asciidoc

```diff
@@ -12,9 +12,25 @@
 
 //tag::notable-breaking-changes[]
 
-//[float]
-//==== Breaking change
+[float]
+====  Environment variables can no longer reference other environment variables
 
-//Description
+Environment variables are still supported in the {beats} configuration.
+However, starting in Version 7.7.0, you can no longer reference an environment
+variable that references another environment variable.
+
+For example, if you set `export NAME=${COMPANY_NAME}`, you cannot use
+`name: ${NAME}` in the configuration.
```

Comment:
this is confusing. If in the shell I use `export NAME=${COMPANY_NAME}`, then `COMPANY_NAME` is resolved immediately and NAME has a constant string. Escaping the `$` will indeed display the problem.

Besides that, environment variables can not contain objects anymore. E.g. `export HOSTS="[a, b, c]"` used to be parsed into an array of 3. This is not the case anymore I think.

Reply:
Can I get help with an example + text that is better then?

Reply:
@urso and @dedemorton You are both correct, neither will result in previous behavior.

The simplest example is an environment variable cannot reference another environment variable or event a reference in the configuration.

So I would go with:

For example, if you have `export FOO=${BAR}` and use `var: ${FOO}` in the configuration, then the configuration will result in `var: "${BAR}"` where `${BAR}` will not be expanded.

Reply:
Sounds good! I'll have to change "foo" and "bar", though, or former editors will come back from the grave to haunt me. :-)